.github/workflows/ci.yml

```diff
@@ -69,8 +69,10 @@ jobs:
         run: composer update --no-progress --no-interaction --prefer-dist --prefer-lowest
         if: ${{ matrix.dependencies == 'lowest' }}
 
-      - name: Run tests
-        run: vendor/bin/phpunit --coverage-clover=coverage.xml
+      - name: Run unit tests
+        run: vendor/bin/phpunit --testsuite unit --coverage-clover=coverage.xml
+      - name: Run phpt tests
+        run: vendor/bin/phpunit --testsuite phpt --no-coverage
```

Comment:
The reason is that the OOM tests OOM because of the coverage information being collected. I'm working on moving the OOM tests to their own folder to only run those without coverage.